airbyte-cdk/python/unit_tests/sources/declarative/retrievers/test_simple_retriever.py

```diff
@@ -273,6 +273,46 @@ def test_get_request_headers(test_name, paginator_mapping, expected_mapping):
                 pass
 
 
+@pytest.mark.parametrize(
```

Comment:
For my education: assuming this thing makes the test below run each time for each of the parameter tuples in the list, right? Neat.

Reply:
Yes, this is the `parametrized tests`, where each tuple element is another case for the test.